src/styles/arduino-dark.css

```diff
@@ -9,6 +9,7 @@
   overflow-x: auto;
   padding: 0.5em;
   background: #282c34;
+  font-family: Hack;
```

Comment:
Sorry, themes may not specify custom fonts.

Reply:
Oh, I was trying my font and I forget to remove...